style.css

```diff
@@ -92,10 +97,227 @@ nav img:hover {
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.works {
+  background-color: #fff;
+  width: 375px;
+  display: grid;
+  grid-template-columns: 324px;
+  grid-template-rows: 237px, 386px, 386px, 386px, 386px, 386px, 386px;
+  padding: 60px 30px;
+}
+
+.content1 {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+}
+
+.content1 h2 {
+  width: 327px;
+  margin-top: 20px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.content1 li {
+  width: 327px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  margin-top: 30px;
+  align-items: center;
+  color: #091e42;
+  text-decoration: none;
+}
+
+.content1 p {
+  width: 327px;
+  height: 147px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-top: 20px;
+  color: #091e42;
+}
+
+.content1 div ul {
+  display: flex;
+  align-items: center;
+  padding: 10px 12px;
+  width: 49px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+}
+
+.properties li {
+  margin-left: 30px;
+}
+
+.content1 div li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 5px 8px;
+  font-size: 15px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+  margin-left: 20px;
+  list-style: none;
+}
+
+button {
+  width: 327px;
+  height: 48px;
+  background-color: #ff6b00;
+  color: #fff;
+}
+
+.content1 button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  font-size: 15px;
+  width: 124px;
+  height: 48px;
+  color: #fff;
+  background-color: #ff6b00;
+}
+
+.cards {
+  background-image: url(Images/Img\ Placeholderbackground\ image.png);
+  background-size: cover;
+  background-position: center;
+  padding-left: 10px;
+}
+
+.cards h2 {
+  width: 295px;
+  height: 81px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.cards p {
+  width: 295px;
+  height: 87px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.cards div ul {
+  display: flex;
+}
+
+.About div h3 {
+  width: 164px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  text-decoration: none;
+}
+
+.About div li {
+  width: 114px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.cards div ul li {
+  width: 71px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  margin-left: 20px;
+  padding: 10px 12px;
+  background-color: rgba(255, 255, 255, 0.24);
+}
+
+.card div ul h3:hover {
+  border: 2px solid rgba(15, 202, 231, 0.747);
+  cursor: pointer;
+}
+
+.About {
+  width: 375px;
+  padding: 0 30px;
+  display: flex;
```

Comment:
- [ ] Well done on implementing the changes by the previous reviewer, however, this issue is still persisting, the first reviewer staged it in his comment but as we can see, it is not solved yet, please take care of making a few touches before we can approve your project.
- [ ] 👀..Check this out ⬇️ 

![divine](https://user-images.githubusercontent.com/77843050/205738509-161dbc81-6284-4abd-a5b0-bc43a6d4298c.gif)